src/System.Data.Odbc/src/Configurations.props

```diff
@@ -2,11 +2,11 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netfx-Windows_NT;
+      netcoreapp2.0-Linux;
```

Comment:
remind me why it can't be `netstandard-Linux` etc? isn't that a pure subset of `netcoreapp2.0` ?

Reply:
netstandard-linux isn't really a thing as it isn't a platform. That would be saying any platform that supports netstandard and runs on linux would be able to consume this library which I think is a bit of a stretch. Lets try to keep netstandard non-RID specific

Reply:
yep